### ISSUE TITLE
Add config

### ISSUE THREAD

**Issue by mooseman1241** [syvita/solitude#7]
Adding a struct for config stuff such as timeout for the stream and session style.

**Comment by mooseman1241** [comment]
Also for setting custom ports for SAM connection if needed

**Comment by linden** [comment]
Could we use TOML files?

**Comment by mooseman1241** [comment]
No, we would use a config enum to configure, we wouldn't use any files
